### PR TITLE
Update postgresql_db.py

### PR DESCRIPTION
**Pull request by caylorme**
##### ISSUE TYPE
- Feature Pull Request
##### COMPONENT NAME

database/postgresql/postgresql_db
##### ANSIBLE VERSION

devel
##### SUMMARY

Added a parameter to allow users to specify synchronous_commit for creation, alteration, or deletion of postgresql databases. This will allow users to set up databases and users before the replication nodes have been implemented.

A common use case of this is if a user wants to create the databases in the same module as installing postgres, but has not yet set up the replication nodes, the transaction will never take place

adding the ability to override synchronous_commit on the transactions

### CODE REVIEW

File: database/postgresql/postgresql_db.py [hunk 1]
```diff
@@ -234,6 +250,7 @@ def main():
             encoding=dict(default=""),
             lc_collate=dict(default=""),
             lc_ctype=dict(default=""),
+            synchronous_commit=dict(default="on", choices=["on", "remote_write", "local", "off"),
```
Comment (resmo) [inline database/postgresql/postgresql_db.py:253]: syntax error, there is a missing `]`

**Comment by resmo** [comment]
Thanks @caylorme for this PR. A maintainer of this module has asked for revisions to this PR. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

File: database/postgresql/postgresql_db.py [hunk 2]
```diff
@@ -79,6 +79,11 @@
       - Character classification (LC_CTYPE) to use in the database (e.g. lower, upper, ...) Must match LC_CTYPE of template database unless C(template0) is used as template.
     required: false
     default: null
+  synchronous_commit:
+    description:
+      - Specifies whether transaction commit will wait for WAL records to be written to disk before the command returns a "success" indication to the client. Valid values are on, remote_write, local, and off. The default, and safe, setting is on.
+    required: false
+    default: true
```
Comment (resmo) [inline database/postgresql/postgresql_db.py:86]: please add `version_added: "2.1"` to this new arg

**Comment by caylorme** [comment]
@resmo : I'm going to close this so we can use PR 3326. This PR contains the changes you've requested, but contains the changes for both modules.

https://github.com/ansible/ansible-modules-core/pull/3326

**Pull request closed without merging**